apps/web/pages/availability/index.tsx

```diff
@@ -129,14 +132,54 @@ const WithQuery = withQuery(trpc.viewer.availability.list as any);
 
 export default function AvailabilityPage() {
   const { t } = useLocale();
+  const searchParams = useSearchParams();
+  const router = useRouter();
+  const pathname = usePathname();
+  const orgBranding = useOrgBranding();
+
+  // Get a new searchParams string by merging the current
+  // searchParams with a provided key/value pair
+  const createQueryString = useCallback(
+    (name: string, value: string) => {
+      const params = new URLSearchParams(searchParams ?? undefined);
+      params.set(name, value);
+
+      return params.toString();
+    },
+    [searchParams]
+  );
   return (
     <div>
       <ShellMain
         heading={t("availability")}
         hideHeadingOnMobile
         subtitle={t("configure_availability")}
-        CTA={<NewScheduleButton />}>
-        <WithQuery success={({ data }) => <AvailabilityList {...data} />} customLoader={<SkeletonLoader />} />
+        CTA={
+          <div className="flex gap-2">
+            {orgBranding && (
+              <ToggleGroup
+                className="hidden md:block"
+                defaultValue={searchParams?.get("type") ?? "mine"}
+                onValueChange={(value) => {
+                  router.push(`${pathname}?${createQueryString("type", value)}`);
+                }}
+                options={[
+                  { value: "mine", label: t("my_availability") },
+                  { value: "team", label: t("team_availability") },
+                ]}
+              />
+            )}
```

Comment:
If we are in an org -> Display toggle to switch between your availability and your teams